vnext/JSI/Shared/PreparedJavaScriptStore.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+namespace facebook::jsi {
+class PreparedJavaScript;
+}
+
+namespace Microsoft::JSI {
+
+/**
+ * TODO (yicyao)
+ */
+class PreparedJavaScriptStore {
+ public:
+  virtual ~PreparedJavaScriptStore() = 0;
```

Comment:
>0 [](start = 39, length = 1)

should this be default?

Reply:
Yes. Will fix this.

---
In reply to: [345464888](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345464888) [](ancestors = 345464888)